packages/react/src/components/Footer/Footer.js

```diff
@@ -17,15 +30,56 @@ class Footer extends React.Component {
    * @returns {object} JSX object
    */
   render() {
+    const { type } = this.props;
+
     return (
-      <footer>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-        {localeSelector ? (
-          <LocaleSelector text="Load the Locale Selector!" />
-        ) : null}
+      <footer
```

Comment:
@photodow can you add in stable selectors here?
https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/docs/stable-selectors.md